### ISSUE TITLE
utils: "format of __builint__.str" is not a built-in

### ISSUE THREAD

**Issue by moylop260** [PyCQA/pylint#1083]
### Steps to reproduce
1. Save the following code in a file:
  ```python
#!/usr/bin/python
from __future__ import print_function

import astroid
from astroid import builder
from pylint.checkers import utils
from pylint.checkers.utils import safe_infer

abuilder = builder.AstroidBuilder()
code_str = '"value: {}".format("value1")'
module = abuilder.string_build(code_str)
node = module.nodes_of_class(astroid.Call).next()
node.func.infer().next()
node_infer = safe_infer(node.func)
print("format infer:", node_infer)
print("Is 'format' built-in?", utils.is_builtin(node_infer))
```

2. Execute the file: `python my_file.py`

### Current behaviour
```bash
format infer: <BoundMethod format of __builtin__.str at 0x4466629456
Is 'format' built-in?: False
```

### Expected behaviour
```bash
format infer: <BoundMethod format of __builtin__.str at 0x4466629456
Is 'format' built-in?: True
```

IMHO the method `format of __builint__.str` should be detect a built-in

### pylint --version output
```bash
pylint 2.0.0,
astroid 1.5.0
Python 2.7.10 (default, Oct 23 2015, 19:19:21)
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)]
```

**Comment by moylop260** [comment]
Fixed using the following change: `s/utils.is_builtin(node_infer)/utils.is_builtin_object(node_infer)/g`
Sorry
